MEME/MEME/Model/ModelReservationView/ModelReservationViewController.swift

```diff
@@ -634,16 +639,23 @@ extension ModelReservationViewController {
                     
                     review.reviewImgDtoList.forEach { imgDto in
                         imageDispatchGroup.enter()
-                        FirebaseStorageManager.downloadImage(urlString: imgDto.reviewImgSrc) { downloadedImage in
-                            if let image = downloadedImage {
-                                images.append(image)
+                        
+                        if let url = URL(string: imgDto.reviewImgSrc) {
+                            let task = URLSession.shared.dataTask(with: url) { data, response, error in
+                                defer { imageDispatchGroup.leave() }
+                                
+                                if let data = data, let image = UIImage(data: data) {
+                                    images.append(image)
```

Comment:
```suggestion
                                if 
                                     let data = data,
                                     let image = UIImage(data: data) 
                                 {
                                    images.append(image)
```

조건이 여러개라면 위와 같이 사용하면 더 좋을 것 같습니다!